api-tests/src/steps/cri-steps.ts

```diff
@@ -336,3 +418,38 @@ When(
     );
   },
 );
+
+function assertNoUnexpectedJarProperties(
+  jarPayload: CriStubResponseJarPayload,
+  expectedNonStandardValues: string[] | undefined = undefined,
+) {
+  for (const key of removeStandardJarValues(Object.keys(jarPayload))) {
+    const expectedValueExists =
+      expectedNonStandardValues === undefined
+        ? false
+        : expectedNonStandardValues.includes(key);
+    assert.equal(
```

Comment:
You can use `assert.ok` if you just want truthiness

Reply:
Thanks, I was surprised `assert.true` didn't exist